ACS_child_allowance_rough_draft.py

```diff
@@ -0,0 +1,178 @@
+import numpy as np
+import pandas as pd
+import microdf as mdf
+
+raw = pd.read_stata(
+    "spm_2018_pu.dta",
+    columns=[
+        "spm_id",
+        "puma",
+        "spm_povthreshold",
+        "spm_resources",
+        "age",
+        "spm_totval",
+        "st",
+        "wt",
+    ],
+)
+print(raw.columns.values)
+
+# VAR LIST: 'spm_id' 'spm_povthreshold' 'spm_resources'
+#  'puma' 'age' 'st' 'spm_totval' 'wt'
+
+person = raw
+person["child"] = np.where(person["age"] < 18, 1, 0)
+spmu = person.groupby(["spm_id", "spm_resources"])[["child"]].sum().reset_index()
+spmu.rename(columns={"child": "spmu_children"}, inplace=True)
+spmu["new_resources"] = spmu["spm_resources"] + 1200 * spmu["spmu_children"]
```

Comment:
```suggestion
ANNUAL_CHILD_ALLOWANCE = 1200
spmu["new_resources"] = spmu.spm_resources + ANNUAL_CHILD_ALLOWANCE * spmu.spmu_children
```
to be more explicit about the constant